sphinx/writers/latex.py

```diff
@@ -446,6 +446,18 @@ def __init__(self, document, builder):
             self.elements['multilingual'] = '%s\n\\setmainlanguage{%s}' % \
                 (self.elements['polyglossia'], self.babel.get_language())
 
+        # detect Chinese language, disable babel/polyglossia
+        # give warnings when no common chinese package found in configuration
+        if builder.config.language:
+            if builder.config.language.split('_')[0] == 'zh':
+                self.elements['babel'] = ''
+                self.elements['polyglossia'] = ''
+                if 'ctex' not in self.elements['usepackages'].lower():
```

Comment:
This is same above.
Isn't there any choices to build Chinese docs without ctex?

In this comment, https://github.com/sphinx-doc/sphinx/pull/3251#issuecomment-267512238, you said CJKutf8 package allows to build Chinese docs.
This warns to the such users. Is this intended?